aws/cloudformation/standalone/gen_ai_curriculum/gen_ai_curriculum.yml.erb

```diff
@@ -0,0 +1,81 @@
+# Gen AI Curriculum Resources
+# This template is manually deployed to create/update Gen AI Curriculum resources.
+# TODO:
+  # Fix hardcoded execution role ARN
+
+AWSTemplateFormatVersion: 2010-09-09
+
+Description: SageMaker resources for Gen AI Curriculum.
+
+Parameters:
+  Environment:
+    Type: String
+    Default: "test"
+    Description: "Environment being deployed to (production or test)"
+  HuggingFaceToken:
+    Type: String
+    Description: "HuggingFace token with read access; required for deploying HuggingFace models."
+  HuggingFaceImageUri:
+    Type: String
+    Description: "HuggingFace Image URI to use for the model."
+Resources:
+<% Config::ENDPOINT_CONFIGS.each do |config| %>
+  <%= config[:model_name] %>Model:
+    Type: AWS::SageMaker::Model
+    Properties:
+      ModelName: !Sub "<%= "#{config[:model_name]}-${Environment}-#{Time.now.to_i}" %>"
+      ExecutionRoleArn: !Sub "arn:aws:iam::${AWS::AccountId}:role/AmazonSageMaker-ExecutionRole-test"
+      PrimaryContainer:
+        Environment:
+          {
+            "HF_MODEL_ID": "<%= config[:hf_model_id] %>",
+            "HUGGING_FACE_HUB_TOKEN": !Sub "${HuggingFaceToken}",
+            "MAX_INPUT_LENGTH": 3000,
+            "MAX_TOTAL_TOKENS": 4096,
+            "SAGEMAKER_CONTAINER_LOG_LEVEL": 20,
+            "SAGEMAKER_REGION": !Sub "${AWS::Region}",
+            "SM_NUM_GPUS": 1,
+          }
+        Image: !Sub "${HuggingFaceImageUri}"
+        Mode: "SingleModel"
+  <%= config[:model_name] %>EndpointConfig:
```

Comment:
Can we add a whitespace line between each resource for easier scanning?

Reply:
Done